lib/Streaming/ByteString.hs

```diff
@@ -56,18 +57,17 @@ module Streaming.ByteString
   , pack             -- pack :: Monad m => Stream (Of Word8) m r -> ByteStream m r
   , unpack           -- unpack :: Monad m => ByteStream m r -> Stream (Of Word8) m r
   , fromLazy         -- fromLazy :: Monad m => ByteString -> ByteStream m ()
-  , toLazy           -- toLazy :: Monad m => ByteStream m () -> m ByteString
-  , toLazy_          -- toLazy' :: Monad m => ByteStream m () -> m (Of ByteString r)
+  , toLazy           -- toLazy :: Monad m => ByteStream m () -> m (Of ByteString r)
+  , toLazy_          -- toLazy_ :: Monad m => ByteStream m () -> m ByteString
```

Comment:
You know what, let's just kill all these signature comments. They're a maintenance burden.

Reply:
That's fine.  Would you like me to do that in this PR?

Reply:
Yes please :)